src/components/MenuItem/MenuItem.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useContext } from "react";
+import { CartContext } from "../../CartContext";
+import "./MenuItem.css";
+
+function MenuItem({ item }) {
+  const [qty, setQty] = useState(1);
+  const { addToCart } = useContext(CartContext);
+
+  const handleAddToCart = () => {
+    for (let i = 0; i < qty; i++) {
```

Comment:
Could it be done by multiplying the price by qty?
It looks like it should be possible to add all items at once, instead of adding them one by one in a loop